charts/redpanda/templates/console/configmap-and-deployment.yaml

```diff
@@ -93,19 +115,22 @@ limitations under the License.
       "kafka" $consoleConfigKafka
       "connect" $connectConfig
 }}
+
 {{ $config := dict
     "Values" (dict
       "console" (dict "config" $consoleConfig)
       "configmap" $consoleConfigmap
+      "secret" $secretConfig
```

Comment:
`secret` values are not used in config map. Why it is added here?

Reply:
The pattern is to add items of the values file, that we are generating from above, regardless if they are used necessarily. This may be the reason why this file is not split into two files?